### DISCUSSION TITLE
如何进行离线插件安装？

### DISCUSSION THREAD

**Discussion by coulis1018** [opensumi/core#1795] [General]
除了通过扩展 插件市场安装插件，是否支持离线安装插件，具体操作步骤是怎样的呢

**Comment by erha19** [comment]
@coulis1018 离线安装目前支持两种方式：

1. 如果是 .vsix 后缀文件，可以手动修改后缀为 .zip 后解压，可将内部的 out 目录作为插件根目录放置在自定义的插件目录下进行安装
2. 如果是常规的本地开发插件，可以直接放置在框架内设置的插件目录进行安装
